File: SERVIDOR CENTRAL/lib/domain/boundary/input/get_attendance_control_list_use_case.go
package input

import "gitlab.com/jorgechavezrnd/servidorseminariogo2/lib/domain/dto/request"

// GetAttendanceControlListUseCase ...
type GetAttendanceControlListUseCase interface {
	GetAttendanceControlList(request request.GetAttendanceControlListRequest)
}

File: SERVIDOR SDR_SCR/lib/domain/boundary/input/get_attendance_control_list_use_case.py
from abc import ABC, abstractmethod
import asyncio
from lib.domain.dto.request.get_attendance_control_list_request import GetAttendanceControlListRequest

class GetAttendanceControlListUseCase(ABC):
    @abstractmethod
    async def getAttendanceControlList(self, request: GetAttendanceControlListRequest): pass